### PR TITLE
SALAMANDER website update

### PR DESCRIPTION
**Pull request by simopier**
Significant update to the SALAMANDER website to:
- Update front page and include logo
- Create documentation for getting started with SALAMANDER, V&V and example page, etc. 
- Update README
- Add list of publications supporting SALAMANDER development
- etc. 

(Ref. #32)

### CODE REVIEW

**Comment by simopier** [comment]
I'm still struggling with a couple issues locally:

> MooseDocs.extensions.bibtex (ForkProcess-12): Unknown BibTeX key: piro2013
MooseDocs.base.renderers (ForkProcess-12): 
RENDER ERROR: Unable to locate a page that ends with the name 'moose/modules/chemical_reactions/index.md'. Did you mean one of the following:
     modules/ray_tracing/index.md
     modules/electromagnetics/index.md
     MooseDocs/extensions/katex.md
     MooseDocs/extensions/bibtex.md
     MooseDocs/extensions/core.md
> /Users/simopa/projects/fenix/doc/content/getting_started/installation.md:
[moose/modules/chemical_reactions/index.md] 

> MooseDocs.base.renderers (ForkProcess-12): 
RENDER ERROR: 'piro2013'
/Users/simopa/projects/fenix/doc/content/getting_started/installation.md:163

Any suggestions on how to handle that? I've tried several versions of `'moose/modules/chemical_reactions/index.md'`, but without sucess.

**Comment by cticenhour** [comment]
@simopier Thanks for this update. I was doing some similar things for MALAMUTE, and FENIX was upcoming on my own list of documentation tasks. 😄 I will likely be able to get to this tomorrow, due to all of the FES proposal reviews and writing going on.

**Comment by cticenhour** [comment]
> Any suggestions on how to handle that? I've tried several versions of 'moose/modules/chemical_reactions/index.md', but without sucess.

Chemical reactions module content is not being pulled in in the `config.yml` file. That's why you're seeing the error. 

EDIT: When you do have it available `chemical_reactions/index.md` will suffice.

**Comment by simopier** [comment]
It's returning a bunch of new errors. I'm not sure how to fix it.

File: doc/content/index.md
Comment (cticenhour) [inline doc/content/index.md]:
```suggestion
!media figures/FENIX_logo_blue.png dark_src=figures/FENIX_logo_white.png style=display:block;margin-left:auto;margin-right:auto;width:40%;
```

You should consider adding the white FENIX logo as the dark mode alternate using the above syntax. I did this on the newest version of the MALAMUTE site, and it's very nice 😄

File: doc/content/index.md
Comment (cticenhour) [inline doc/content/index.md]:
```suggestion
FENIX is designed as an open-source, fully integrated, multiphysics, multiscale, Nuclear Quality Assurance Level 1 (NQA-1) compliant framework facilitating 3D, high-fidelity fusion system modeling.
```

File: doc/content/index.md
Comment (cticenhour) [inline doc/content/index.md]: I have slightly changed my mind since the technical meeting, in light of the fact that we have a `User Notes` page in this PR. Let's move this section there to kick-off use of that page!

File: doc/content/index.md
Comment (cticenhour) [inline doc/content/index.md]: This INL logo should be moved to the bottom of the page, and then logos for NCSU, UIUC, and the UKAEA should be added alongside.

File: doc/content/publications.md
Comment (cticenhour) [inline doc/content/publications.md]:
```suggestion
!! The below commented out sections should be added in as publications are created and available

!! ## Journal Articles

## Conference Proceedings

!bibtex list bib_files=articles_using_fenix.bib
```

I am thinking that we should create a conference-specific list where the PBNC (and TOFE, and others...) papers/presentations will live, and leave the journal article section hidden for now. If we go with this, I also suggest we change the file name to `proceedings_using_fenix.bib`.

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
# FENIX Verification, Validation, and Example Cases
```

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
For [software quality assurance (SQA)](sqa/index.md) purposes, FENIX undergoes verification, validation, and benchmarking. The FENIX development team defines these terms as:

- +Verification+: Comparing FENIX predictions against analytical solutions in different conditions, which are often simple cases.
- +Validation+: Comparing FENIX predictions against experimental data.
- +Benchmarking+: Comparing FENIX predictions against other codes.

Note that in addition to monitoring FENIX performance and reproducibility in verification and validation cases, the effects of changes made to FENIX are tracked. A series of automated tests are performed via continuous integration using [CIVET](https://civet.inl.gov/repo/1108) to help identify any changes in FENIX's predictions, therefore ensuring stability and robustness.

Example cases have also been added, independent of the SQA cases shown below, which showcase existing FENIX capabilities and demonstrate potential applications. In all cases, the existing input files should allow users to leverage prior work.
```

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
FENIX couples with other MOOSE-based applications, such as [TMAP8](https://mooseframework.inl.gov/tmap8) or [Cardinal](https://cardinal.cels.anl.gov/). These projects have their own documentation regarding SQA and examples, which can be found at the following locations:

- [TMAP8 Verification and Validation](https://mooseframework.inl.gov/tmap8/verification_and_validation)
- [Cardinal Tutorial](https://cardinal.cels.anl.gov/tutorials/index.html)
```

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
!alert construction title=Under development - no verification cases are available yet
```

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
!alert construction title=Under development - no validation cases are available yet
```

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
!alert construction title=Under development - no benchmarking cases are available yet
```

File: doc/content/verification_validation_examples/index.md
Comment (cticenhour) [inline doc/content/verification_validation_examples/index.md]:
```suggestion
!alert construction title=Under development - no example cases are available yet
```

File: doc/content/getting_started/installation.md
Comment (cticenhour) [inline doc/content/getting_started/installation.md]:
```suggestion
- [Windows (experimental)](https://mooseframework.inl.gov/getting_started/installation/windows.html)
```

Windows 11 is now a thing, so the MOOSE docs were updated to reflect this.

File: doc/content/getting_started/installation.md
Comment (cticenhour) [inline doc/content/getting_started/installation.md]:
```suggestion
```

This is where the chemical reactions failure was coming from. Since we're not using that module in FENIX (at the moment at least), it should be removed.

File: doc/content/getting_started/using_fenix.md
Comment (cticenhour) [inline doc/content/getting_started/using_fenix.md]:
```suggestion
cases can be found [on the FENIX verification, validation, and example page](verification_validation_examples/index.md).
```

File: doc/content/getting_started/using_fenix.md
Comment (cticenhour) [inline doc/content/getting_started/using_fenix.md:49]:
```suggestion
!style halign=left
As FENIX is developed and more capabilities are added, the tests directory will continue to
```

File: doc/config.yml
Comment (cticenhour) [inline doc/config.yml]:
```suggestion
```

Not needed if that installation tip on Thermochimica is removed.

File: doc/config.yml
Comment (cticenhour) [inline doc/config.yml:49]:
```suggestion
                Code Standards: sqa/fenix_scs.md
                Code Coverage: https://mooseframework.inl.gov/fenix/docs/coverage
```

This URL isn't live yet, but we generate coverage on every merge to main. CIVET supposedly publishes the results here (see https://civet.inl.gov/job/2210582/). Likely just need to make the spot for it on the server public or re-route it to `https://fenix-dev.hpc.inl.gov/docs/coverage`. @loganharbour can you advise and assist with the best route forward for now? FENIX should be open-source on the order of months.

File: README.md
Comment (cticenhour) [inline README.md]:
```suggestion
electromagnetics, heat conduction, scalar transport, thermal hydraulics, computational fluid dynamics,
```

**Review by cticenhour — CHANGES_REQUESTED** [review]
Good content! The Thermochimica alert removal and reverting the `config.yml` content change should address the test failures.

File: doc/content/index.md
Comment (simopier) [reply doc/content/index.md]: 🤯 
Niiiiiiiiiiiiiiiiice!!

File: doc/config.yml
Comment (simopier) [reply doc/config.yml:49]: I added this suggestion, but leaving this discussion as unresolved since @loganharbour needs to jump in.

File: doc/content/index.md
Comment (cticenhour) [inline doc/content/index.md]:
```suggestion
!media large_media/framework/inl_blue.png style=display:block;margin-left:auto;margin-right:auto;width:30%;

!row! style=display:inline-flex;
!col! small=12 medium=4 large=2

!media figures/ncstate-brick-2x2-red.png style=width:100%;display:block;

!col-end!

!col! small=12 medium=4 large=2

!media figures/University-Wordmark-Full-Color-RGB.png dark_src=figures/University-Wordmark-Reversed-Orange-RGB.png style=width:100%;display:block;

!col-end!

!col! small=12 medium=4 large=2

!media figures/UK_Atomic_Energy_Authority_logo.svg dark_src=figures/UK-AEA_WHITE_SML_AW.png style=width:100%;display:block;

!col-end!
!row-end!
```

When clicked on, the render for these kind of goes "wild", with the images darting to the left. Let's see how the this suggested change looks; it uses the [Layout extension](https://mooseframework.inl.gov/python/MooseDocs/extensions/layout.html) to let each one have its own "dedicated space" in the window without needing to manually set all of the CSS yourself. I also fixed a typo for the "light mode" UKAEA logo - it was displaying UIUC instead.

File: doc/content/index.md
Comment (cticenhour) [reply doc/content/index.md]: Looks good when glancing at it in "dark mode", but there are several issues if the revised version is clicked on, or the browser is in "light mode". See my comment below. Resolving this comment.

File: doc/content/index.md
Comment (cticenhour) [inline doc/content/index.md]:
```suggestion
!media large_media/framework/inl_blue.png link=https://www.inl.gov style=display:block;margin:auto;width:30%;

&nbsp;

!row! style=display:inline-flex;
!col! small=12 medium=4 large=3

!media figures/ncstate-brick-2x2-red.png link=https://www.ncsu.edu/ style=width:100%;display:block;

!col-end!

!col! small=12 medium=4 large=3

!media figures/University-Wordmark-Full-Color-RGB.png link=https://illinois.edu/ dark_src=figures/University-Wordmark-Reversed-Orange-RGB.png style=width:100%;padding:20px 0;display:block;

!col-end!

!col! small=12 medium=4 large=3

!media figures/UK_Atomic_Energy_Authority_logo.svg link=https://ccfe.ukaea.uk/ dark_src=figures/UK-AEA_WHITE_SML_AW.png style=width:50%;display:block;
```

After testing all of this locally, this configuration looks the best so far. This change:

- Shortens some CSS for the INL logo
- Adds links to all logos (a click no longer zooms in on the image, but instead goes to the URL provided by `link`)
- Makes the NCSU, UIUC, and UKAEA logos a bit more proportional and centered (both horizontally and vertically) with each other.

File: doc/config.yml
Comment (cticenhour) [inline doc/config.yml]:
```suggestion
                #Code Coverage: https://mooseframework.inl.gov/fenix/docs/coverage
```

I took a peek at the CIVET scripts, and it seems (at first glance....could have missed something late yesterday) that the current HTML generation script for coverage assumes the sync URL based on the location of the repository rather than whether it is `private` or not. I am suggesting we comment this out for now and then restore it when we make the site fully public. 

I have made an issue (#35). 

This should be the last item before we merge this - I think this version of the website looks good!

File: doc/config.yml
Comment (cticenhour) [reply doc/config.yml:49]: Will revisit as part of addressing #35. Resolving.

**Review by cticenhour — APPROVED** [review]

**Comment by cticenhour** [comment]
All comments addressed. Good changes! Will merge when the pipeline is all green.

**Pull request merged**
